spec/lib/bank_account/savings_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'bank_account'
+
+describe Bank::SavingsAccount do
+  context "attributes" do
+    let(:savings) {Bank::SavingsAccount.new("keep_it", 100)}
+
+    it "responds to :id" do
+      expect(savings).to respond_to :id
+    end
+
+    it "resonds to :balance" do
+      expect(savings).to respond_to :balance
+    end
+
+    it "has id 'keep_it'" do
+      expect(savings.id).to eq("keep_it")
+    end
+
+    it "has an initial balance of 100" do
+      expect(savings.balance).to eq(100)
+    end
+
+    it "returns 48 for a withdrawal of 50" do
+      expect(savings.withdraw(50)).to eq(48)
+    end
+
+    context "accruing interest" do
+      let(:more_savings) {savings.add_interest(10)}
```

Comment:
You also might not need a separate context for this. Has something super wild happened in your setup for this test that differs from the other tests?